python/cudf_polars/cudf_polars/callback.py

```diff
@@ -32,6 +33,21 @@
 __all__: list[str] = ["execute_with_cudf"]
 
 
+_SUPPORTED_PREFETCHES = {
+    "column_view::get_data",
+    "mutable_column_view::get_data",
+    "gather",
+    "hash_join",
+}
+
+
+def _env_get_int(name, default):
+    try:
+        return int(os.getenv(name, default))
+    except (ValueError, TypeError):
+        return default
+
+
 @cache
 def default_memory_resource(device: int) -> rmm.mr.DeviceMemoryResource:
```

Comment:
```suggestion
def default_memory_resource(device: int) -> tuple[rmm.mr.DeviceMemoryResource, bool]:
```

No need to do this if we configure the prefetching within the MR construction.

Reply:
No more needed as prefetching is done in this method.